src/patternfly/components/Expandable/expandable.scss

```diff
@@ -0,0 +1,38 @@
+.pf-c-expandable {
+  // Expandable Toggle
+  --pf-c-expandable__toggle--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-expandable__toggle--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-expandable__toggle--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-expandable__toggle--PaddingLeft: 0;
+  --pf-c-expandable__toggle--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-expandable__toggle--Color: var(--pf-global--link--Color);
+
+  // Expandable Toggle Icon
+  --pf-c-expandable__toggle-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-expandable__toggle-icon--Color: var(--pf-global--Color--100);
+
+  // Expandable Expanded
+  --pf-c-expandable__content--MarginTop: var(--pf-global--spacer--xl);
+
+  &.pf-m-expanded {
+    .pf-c-expandable__toggle-icon {
+      transform: rotate(90deg);
+    }
+  }
+
+  .pf-c-expandable__toggle {
+    padding: var(--pf-c-expandable__toggle--PaddingTop) var(--pf-c-expandable__toggle--PaddingRight) var(--pf-c-expandable__toggle--PaddingBottom) var(--pf-c-expandable__toggle--PaddingLeft);
+    font-weight: var(--pf-c-expandable__toggle--FontWeight);
+    color: var(--pf-c-expandable__toggle--Color);
+    border: none;
```

Comment:
should we add....

```
  --pf-c-expandable__toggle--Color: var(--pf-global--link--Color--hover);
```

Reply:
ah yeah theres no hover/focus styles specified. If we add this, @mceledonia should there be an active/focus state too?